scripts/src/chartrepomanager/chartrepomanager.py

```diff
@@ -24,12 +24,24 @@
 from chartrepomanager import indexannotations
 from signedchart import signedchart
 from pullrequest import prartifact
+from reporegex import matchers
 from tools import gitutils
 
 
 def get_modified_charts(api_url):
+    """Reads a PR's modified files to get submission details and exits if none are found.
+
+    Args:
+        api_url: URL to the API representation of a pull request.
+          E.g. https://api.github.com/repos/openshift-helm-charts/charts/pulls/1
+
+    Returns:
+        The category, organization, chart name, and version.
+    """
```

Comment:
Would you be fine with removing this docstring ? See https://github.com/openshift-helm-charts/development/pull/278

Reply:
Fixed